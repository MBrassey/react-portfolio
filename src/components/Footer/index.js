import React from "react";

function Footer() {
  const icons = [
    {
      name: "fab fa-github",
      link: "https://github.com/MBrassey",
    },
    {
      name: "fab fa-linkedin",
      link: "https://www.linkedin.com/in/mbrassey/",
    },
    {
      name: "far fa-envelope",
      link: "mailto:matt@brassey.io",
    },
    {
      name: "fas fa-file-alt",
      link: "https://docs.google.com/document/d/1T7uHv2RcH_wzERwJKoahqh9ODKdaJ48ci40nKEBViNY/",
    },
  ];

  return (
    <footer className="footer">
      {icons.map((icon) => (
        <a
          href={icon.link}
          key={icon.name}
          target="_blank"
          rel="noopener noreferrer"
        >
          <i className={icon.name}></i>
        </a>
      ))}
    </footer>
  );
}

export default Footer;