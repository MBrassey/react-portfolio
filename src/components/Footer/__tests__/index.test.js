import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import React from 'react';

import Footer from '..';

afterEach(cleanup);

describe('Footer component', () => {
  it('renders', () => {
    render(<Footer />);
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<Footer />);

    expect(asFragment()).toMatchSnapshot();
  });
})
