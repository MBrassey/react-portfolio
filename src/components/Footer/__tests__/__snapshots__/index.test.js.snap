// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <a
      href="https://github.com/MBrassey"
      rel="noopener noreferrer"
      target="_blank"
    >
      <i
        class="fab fa-github"
      />
    </a>
    <a
      href="https://www.linkedin.com/in/mbrassey/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <i
        class="fab fa-linkedin"
      />
    </a>
    <a
      href="mailto:matt@brassey.io"
      rel="noopener noreferrer"
      target="_blank"
    >
      <i
        class="far fa-envelope"
      />
    </a>
  </footer>
</DocumentFragment>
`;
